Server/TestPLan_template_system_tagging.md

```diff
@@ -0,0 +1,24 @@
+Previous requirements: use https and enable encryption.
+
+
+
+| Test Case                                | Expected Result                          | Result | Related Comment |
+| ---------------------------------------- | ---------------------------------------- | ------ | --------------- |
+| Using admin share a file 'File.txt' with user1, with user1 share it with user2. As admin put a tag on 'File.txt'. Edit the tag with user1 and with user2. | User1 and user2 can edit the tag and the changes affect all users. |        |                 |
+| Put a few tags on a file when there are several tags previously assigned. Remove some as well. | A list of possible tags appear on the sidebar when clicking on the textfield. They can be removed and added using checkboxes. |        |                 |
```

Comment:
I suggest you use the term "assign" and "unassign" instead of added/removed, to avoid confusing it with "creating" and "deleting" tags.